ext/spl/spl_fixedarray.stub.php

```diff
@@ -16,7 +16,7 @@ public function count() {}
     public function toArray() {}
 
     /** @return SplFixedArray */
-    public static function fromArray(array $array, bool $save_indexes = true) {}
+    public static function fromArray(array $array, bool $preserveIndexes = true) {}
```

Comment:
Doesn't this parameter have the same purpose as `$preserve_keys` in e.g. `iterator_to_array`? Or the reason of the key-index distinction is because this one only deals with numeric keys? Then I can imagine the following options:
- `$preserveIndexes`
- `$preserveIndices` - at least this is the plural form of index I heard most often
- `$preserveNumericKeys`

I have a slight preference for the last idea, but if you find it too long, please feel free to use a reduced version

Reply:
I went with just `$preserveKeys` here. The difference to other usages is that string keys here will throw, because they can't be preserved, but fundamentally this parameter is the same as other `$preserve_keys` usages.

Reply:
Perfect!